src/app/settings/imprint/imprint.page.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, OnInit } from '@angular/core'
+
+@Component({
+  selector: 'app-imprint',
+  templateUrl: './imprint.page.html',
+  styleUrls: ['./imprint.page.scss'],
+})
+export class ImprintPage implements OnInit {
+  constructor() {}
+
+  ngOnInit() {}
```

Comment:
If not needed, the implementation of `OnInit` could be removed.

Reply:
Resolved in 42e9b80